bugbug/model.py

```diff
@@ -399,8 +421,9 @@ def train(self, importance_cutoff=0.15, limit=None):
 
         feature_names = self.get_human_readable_feature_names()
         if self.calculate_importance and len(feature_names):
-            explainer = shap.TreeExplainer(self.clf)
-            shap_values = explainer.shap_values(X_train)
+            explainer = shap.TreeExplainer(self.clf.named_steps["estimator"])
+            _X_train = get_transformer_pipeline(self.clf).transform(X_train)
```

Comment:
I imagine this is pretty bad in terms of memory, as we are duplicating the data. I can't think of an alternative solution though

Reply:
Yes, but the peak won't be higher than when we fit the clf pipeline.

Reply:
I plan to simplify the train() method and remove unnecessary data duplicates where possible.